app/src/main/java/com/glia/exampleapp/MainFragment.java

```diff
@@ -39,42 +33,34 @@ public View onCreateView(@NonNull LayoutInflater inflater,
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         NavController navController = NavHostFragment.findNavController(this);
-        view.findViewById(R.id.settings_button).setOnClickListener(view1 -> {
-            navController.navigate(R.id.settings);
-        });
-        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(requireContext());
-        /* https://salemove.atlassian.net/browse/MUIC-362
+        view.findViewById(R.id.settings_button).setOnClickListener(view1 -> navController.navigate(R.id.settings));
+        /* todo https://salemove.atlassian.net/browse/MUIC-362
```

Comment:
Not todo - task that removed it

Reply:
is this comment expected to be there forever or is it assumed to be removed one day?

Reply:
moreover, should we have refs to Jira in open source code?

Reply:
<img width="842" alt="Screenshot 2022-03-18 at 12 32 06" src="https://user-images.githubusercontent.com/49229744/158987278-cdf54bb6-5200-4548-b9d2-69f7250dd776.png">
Well we are having them in pull requests which are visible aswell

Guess just removing the whole comment would be better approach here